package.json

```diff
@@ -48,16 +48,10 @@
   },
   "prettier": "@github/prettier-config",
   "devDependencies": {
-    "@actions/core": "^1.2.7",
     "@actions/github": "^4.0.0",
-    "@babel/parser": "^7.15.3",
-    "@babel/traverse": "^7.15.0",
-    "@babel/types": "^7.15.0",
     "@changesets/changelog-github": "^0.3.0",
     "@changesets/cli": "^2.22.0",
     "@github/prettier-config": "^0.0.4",
-    "@octokit/rest": "^18.5.3",
```

Comment:
Have a feeling that this one is used in the diffing in CI, but that can arguably be installed on the runner from the workflow directly.